pyg_lib/csrc/sampler/cpu/random_walk_kernel.cpp

```diff
@@ -0,0 +1,53 @@
+#include <torch/torch.h>
+
+#include "utils.h"
+
+namespace pyg {
+namespace sampler {
+
+namespace {
+
+torch::Tensor random_walk_kernel(const torch::Tensor& rowptr,
+                                 const torch::Tensor& col,
+                                 const torch::Tensor& seed,
+                                 int64_t walk_length,
+                                 double p,
```

Comment:
looks like we haven't use p, q to control the randomwalk yet.

Reply:
Yes, it is captured in the `TORCH_CHECK`. Will follow in a separate PR. This PR is mostly intended to get the pipeline going :)